src/main/java/victor_murashev/api/petstore/PetstoreBaseRequest.java

```diff
@@ -0,0 +1,36 @@
+package victor_murashev.api.petstore;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.hamcrest.Matcher;
+import org.hamcrest.Matchers;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class PetstoreBaseRequest {
```

Comment:
don't need to create another base request class

Reply:
@dgrimaliuc-crunchyroll Done, please check now.